.cirrus.yml

```diff
@@ -179,13 +180,12 @@ build_and_release_dotnet_task:
     - source cirrus-env QA
     - PowerShell -NonInteractive -NoProfile -File "${PIPELINE_SCRIPTS_DIR}\\build_and_pack_solution.ps1"
     - PowerShell -NonInteractive -NoProfile -File "${PIPELINE_SCRIPTS_DIR}\\extract_project_version.ps1"
-  unit_test_script:
```

Comment:
Moved the UT and IT test execution to the same file not to evaluate if signing should be done for both separately.